src/ir/module-splitting.cpp

```diff
@@ -542,7 +543,7 @@ void ModuleSplitter::setupTablePatching() {
       placeholder->base = std::to_string(index);
       placeholder->name = Names::getValidFunctionName(
         primary, std::string("placeholder_") + placeholder->base.toString());
-      placeholder->hasExplicitName = false;
+      placeholder->hasExplicitName = true;
       placeholder->type = secondaryFunc->type;
```

Comment:
Can the `module-splitting` changes be in a separate PR? I believe only wasm-split uses those so it is not relevant for wasm-merge.

Reply:
Or was the idea to handle all `hasExplicitName` across the codebase? If so then that might make sense but let's remove "wasm-merge" from the PR title.